src/models/config.py

```diff
@@ -53,6 +60,14 @@ def all_families():
                 combined_hash[family] += [config]
         return combined_hash
 
+    @lru_cache()
+    def all_groups():
+        combined_hash = {}
+        groups_list = groups.list()
```

Comment:
How is `all_groups` different to `groups.list()`?

Why do we need to poll every group upfront?